lib/api/collection.js

```diff
@@ -1,7 +1,12 @@
 const { NIL } = require("uuid")
 const { convert } = require('html-to-text');
 const semver = require('semver')
-const dateFormat = require("dateformat")
+const dynamicImport = async (modulePath) => {
```

Comment:
Can you explain why this is necessary?

Reply:
Because we were getting this error:

```
const dateFormat = require("dateformat")
          ^
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/makz/workCS/new-swift/swift-package-registry/node_modules/dateformat/lib/dateformat.js from /Users/makz/workCS/new-swift/swift-package-registry/lib/api/collection.js not supported.
Instead change the require of dateformat.js in /Users/makz/workCS/new-swift/swift-package-registry/lib/api/collection.js to a dynamic import() which is available in all CommonJS modules.
```